app/templates/merchant/view_listings.html

```diff
@@ -54,12 +54,91 @@
 {% endblock %}
 
 {% block after %}
+
+    <script src="{{ url_for('static', filename='merchant.js') }}"></script>
+
     <script type="text/javascript">
-        $(document).ready(function () {
-            $('#toggle-advanced').click(function() {
-                $('.advanced-search').toggle();
+        $('#toggle-advanced').click(function() {
```

Comment:
That's a good point. I was trying to think of why we'd still need it, but that's a good enough reason for me.